### PR TITLE
Update user story and use case

### PR DESCRIPTION
**Pull request by Imranr2**
Tackles part of https://github.com/AY2122S1-CS2103T-W16-2/tp/issues/206

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -910,11 +915,28 @@ Priorities: High - (must have), Medium - (nice to have), Low -  (unlikely to hav
 **Extensions**
```
Comment (EuniceSim142) [inline docs/DeveloperGuide.md:916]: Can you add the duplicate order extension here

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -976,10 +1013,128 @@ Priorities: High - (must have), Medium - (nice to have), Low -  (unlikely to hav
...
+
+      Use case ends.
+
+**Use case: UC18 Add recipe**
+
+**MSS**
+
+1. Home chef chooses to add a recipe.
+2. Home chef enters details in required format.
+2. BTBB displays new recipe.
```
Comment (EuniceSim142) [inline docs/DeveloperGuide.md:1026]:
```suggestion
2. BTBB displays the new recipe.
```

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -976,10 +1013,128 @@ Priorities: High - (must have), Medium - (nice to have), Low -  (unlikely to hav
...
+
+**Extensions**
+
+* 2a. Recipe entered already exists in the system.
+    * 2a1. BTBB informs Home Chef that recipe already exists.
+
+      Use Case ends.
+
+* 2b. BTBB detects an error in the entered data.
+ 	* 2b1. BTBB informs Home Chef why addition of recipe did not happen.
```
Comment (EuniceSim142) [inline docs/DeveloperGuide.md:1038]: Standardise with the order use cases

```suggestion
 	* 2b1. BTBB informs Home Chef why addition of recipe failed.
```

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -976,10 +1013,128 @@ Priorities: High - (must have), Medium - (nice to have), Low -  (unlikely to hav
...
+
+      Use Case ends.
+
+**Use case: UC20 Delete recipe**
+
+**MSS**
+
+1. Home chef chooses to delete a recipe.
+2. Home chef enters details in required format.
+3. BTBB deletes recipe.
```
Comment (EuniceSim142) [inline docs/DeveloperGuide.md:1065]: Standardise with the other use cases
```suggestion
3. BTBB informs Home Chef that recipe has been deleted.
```

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -976,10 +1013,128 @@ Priorities: High - (must have), Medium - (nice to have), Low -  (unlikely to hav
...
+
+   Use case ends.
+
+**Extensions**
+
+* 2a. BTBB detects an error in the entered data.
+    * 2a1. BTBB informs Home Chef why deletion did not happen.
```
Comment (EuniceSim142) [inline docs/DeveloperGuide.md:1072]:
```suggestion
    * 2a1. BTBB informs Home Chef why deletion of recipe failed.
```

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -976,10 +1013,128 @@ Priorities: High - (must have), Medium - (nice to have), Low -  (unlikely to hav
...
+* 2b. BTBB detects an error in the entered data.
+    * 2b1. BTBB informs Home Chef why editing of recipe failed.
+
+      Use case ends.
+
+**Use case: UC23 View revenue for each month for past 12 months**
+
+**MSS**
+
+1. Home chef wants to view revenue earned for each month for past 12 months.
```
Comment (EuniceSim142) [inline docs/DeveloperGuide.md:1112]:
```suggestion
1. Home chef wants to view revenue earned each month for the past 12 months.
```

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -976,10 +1013,128 @@ Priorities: High - (must have), Medium - (nice to have), Low -  (unlikely to hav
...
+2. Home chef switches to statistics tab.
+3. BTBB displays revenue for each month for the past 12 months.
```
Comment (EuniceSim142) [inline docs/DeveloperGuide.md:1114]: Do you think it's better to mention the type of chart (UC24 and UC25 also)?
```suggestion
3. BTBB displays revenue earned each month for the past 12 months in a bar chart.
```

File: docs/DeveloperGuide.md [hunk 8]
```diff
@@ -976,10 +1013,128 @@ Priorities: High - (must have), Medium - (nice to have), Low -  (unlikely to hav
...
+
+   Use case ends.
+
```
Comment (EuniceSim142) [inline docs/DeveloperGuide.md:1117]: Should we have an extension for tab command failure or do you think it's unnecessary?

**Review by EuniceSim142 — COMMENTED** [review]
Mostly formatting and grammatical changes.

Can you standardise either Home Chef or Home chef (one is used in MSS and the other is used in Extensions – just choose one and stick with it).

Also, can you make these changes as well:
- [x] Chef enter**s** details (line 768)
- [x] add **an** ingredient (line 816)
- [x] displays **the** new (line 818)
- [x] add **an** order (line 909)
- [x] displays **the** new order (line 911)
- [x] change to: BTBB informs Home Chef that order has been deleted. (line 945)

File: docs/DeveloperGuide.md (hunk 8 above)
Comment (Imranr2) [reply docs/DeveloperGuide.md:1117]: I don't think its necessary

**Review by EuniceSim142 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**